Workbooks/CosmosDb/Overview/Overview.workbook

```diff
@@ -153,1089 +149,1135 @@
+                  "operator": "is Empty",
+                  "rightValType": "param",
+                  "resultValType": "static",
+                  "resultVal": "false"
+                }
+              },
+              {
+                "condition": "else result = 'true'",
+                "criteriaContext": {
+                  "operator": "Default",
+                  "rightValType": "param",
+                  "resultValType": "static",
+                  "resultVal": "true"
+                }
+              }
+            ]
           },
           {
-            "cellValue": "selectedTab",
-            "linkTarget": "parameter",
-            "linkLabel": "Capacity",
-            "subTarget": "Capacity",
-            "style": "link"
+            "id": "9aab8378-c97c-4b28-b034-7538d6fdfffa",
+            "version": "KqlParameterItem/1.0",
+            "name": "ResourceName",
+            "type": 1,
+            "description": "Used for the 'No Subscriptions' Workbook template",
+            "value": "Cosmos DB",
+            "isHiddenWhenLocked": true,
+            "criteriaData": [
+              {
+                "condition": "else result = 'Cosmos DB'",
+                "criteriaContext": {
+                  "operator": "Default",
+                  "rightValType": "param",
+                  "resultValType": "static",
+                  "resultVal": "Cosmos DB"
+                }
+              }
+            ],
+            "timeContext": {
+              "durationMs": 0
+            },
+            "timeContextFromParameter": "TimeRange"
           },
           {
-            "cellValue": "selectedTab",
-            "linkTarget": "parameter",
-            "linkLabel": "Operations",
-            "subTarget": "Operation",
-            "style": "link"
+            "id": "9fac27da-f78b-466f-ab1f-d4b5a8be2d31",
+            "version": "KqlParameterItem/1.0",
+            "name": "ImageUrl",
+            "type": 1,
+            "description": "Used for the 'No Subscriptions' workbook template",
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 0
+            },
+            "timeContextFromParameter": "TimeRange"
           }
-        ]
+        ],
+        "style": "above",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
       },
-      "name": "links-tabs",
+      "name": "cosmosDB account parameters",
       "styleSettings": {
-        "margin": "-15px 0px -15px 0px"
+        "margin": "15px 0 0 0"
       }
     },
     {
-      "type": 1,
+      "type": 12,
       "content": {
-        "json": "### Start editing the Overview section here"
+        "version": "NotebookGroup/1.0",
+        "groupType": "template",
+        "loadFromTemplateId": "/subscriptions/8980832b-9589-4ac2-b322-a6ae6a97f02b/resourceGroups/oychan-rg/providers/microsoft.insights/workbooks/4297e410-7054-4e23-bacc-5d8418698154",
```

Comment:
TODO: change this to the community one upon check in

Reply:
will be simpler to just check in the no subscriptions content one as its own single PR so you can reference it in this PR

Reply:
Yeah will do that first, I'll make a new PR with this change